crypto/evp/digest.c

```diff
@@ -559,18 +564,39 @@ const OSSL_PARAM *EVP_MD_settable_ctx_params(const EVP_MD *md)
 
 const OSSL_PARAM *EVP_MD_CTX_settable_params(EVP_MD_CTX *ctx)
 {
+    EVP_PKEY_CTX *pctx = ctx->pctx;
+
     if (ctx != NULL
```

Comment:
This check is kind of moot since `ctx` has been dereferenced already.
Coverity will complain I'm sure.

Reply:
Fixed.